pvlib/test/test_pvsystem.py

```diff
@@ -1113,7 +1113,7 @@ def test_pvsyst_celltemp_with_index():
     irrads = pd.Series([0, 500, 0], index=times)
     winds = pd.Series([10, 5, 0], index=times)
 
-    pvtemps = pvsystem.pvsyst_celltemp(irrads, winds, temps)
+    pvtemps = pvsystem.pvsyst_celltemp(irrads, temps, winds)
```

Comment:
Keyword args should be tested as keyword args rather than positional. So `wind_speed=winds` here and below.